E1-basic-exercises/src/main/java/com/redelles/e1/E1cMainProgram.java

```diff
@@ -10,7 +11,24 @@
     public static void main(String[] args) throws IOException {
         System.out.println("Title");
         System.out.println("Introduce the first number");
+        Scanner scan1 = new Scanner(System.in);
+        String n1 = scan1.next();
         //You should search in internet to see how to input numbers from keyboard.
         System.out.println("Introduce the second number");
+        Scanner scan2 = new Scanner(System.in);
+        String n2 = scan2.next();
+
+        Integer x1 = Integer.parseInt(n1);
+        Integer x2 = Integer.parseInt(n2);
+        // Error control
+        if (x1 >= x2){
+            System.out.println("First number should be less than second, try again");
```

Comment:
Try to don't finish the program if the user has input a wrong information, try to show the error and ask again